### ISSUE TITLE
Control theory

### ISSUE THREAD

**Issue by rustyoz** [inkonova/cleanflight#1]
Hi, 
I was looking through your code to gain an insight into how you use the rc pitch channel to control the tilt angle but from what i can gather it is still fed into the pitch PID controller. Is the intended outcome that the copter still pitch in the intended direction of movement but that the tilting rotors provide most of the acceleration, instead of trying to maintain a level body independent of the rotor tilt angle?

**Comment by MauroMombelli** [comment]
hi,

we give full control over this using the pitch divider, see https://github.com/inkonova/cleanflight/blob/tilting-1.10.0/src/main/mw.c#L846

**Comment by rustyoz** [comment]
So you use a self leveling mode with a low proportion of pitch or zero fed into the pitch PID?

Is this only active in one direction of the pitch command? ie pitching forward is scaled down, but pitching back to brake isn't?
https://github.com/inkonova/cleanflight/blob/tilting-1.10.0/src/main/mw.c#L851

**Comment by MauroMombelli** [comment]
no, we control the divider with a potentiometer (tiltArm.channel), if it is
LOWER than mid range the pitch divider is used.

meanwhile the user input PITCH is saved into tmpPitch witch is used for
later elaboration to find out how much move the servo.

2016-04-07 2:21 GMT+02:00 Russell Oliver notifications@github.com:
